torax/tests/test_data/test_iterhybrid_predictor_corrector_impurity_radiation.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Identical to test_iterhybrid_predictor_corrector but simple impurity radiation."""
+
+import copy
+
+from torax.tests.test_data import test_iterhybrid_predictor_corrector
+
+CONFIG = copy.deepcopy(test_iterhybrid_predictor_corrector.CONFIG)
+
+CONFIG['sources']['impurity_radiation_heat_sink'] = {}
```

Comment:
nit: while there is a default, in practice users will almost always set the specific fraction. So maybe good to demonstrate that with adding the runtime_param here.